CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -176,6 +177,9 @@ import Vision
         regionOfInterestLabel.layer.cornerRadius = self.regionCornerRadius
         regionOfInterestLabel.layer.borderColor = UIColor.white.cgColor
         regionOfInterestLabel.layer.borderWidth = 2.0
+
+        self.videoFeed.setup(captureDelegate: self, completion: { success in })
```

Comment:
I'd like to see a comment about why this is here with a pointer to either apple docs or apple example code that does the same thing.